src/isar/modules.py

```diff
@@ -131,3 +131,16 @@ def provide_transform(self, map_config_reader: MapConfigReader) -> Transformatio
             config.get("maps", "eq_robot_default_map_name")
         )
         return Transformation(map_config=map_config)
+
+
+modules: dict = {
+    "coordinate_module": {"default": CoordinateModule},
+    "queues_module": {"default": QueuesModule},
+    "reader_module": {"default": ReaderModule},
+    "request_handler_module": {"default": RequestHandlerModule},
+    "robot_module": {"default": RobotModule},
+    "service_module": {"local": ServiceModule, "echo": ServiceModule},
+    "state_machine_module": {"default": StateMachineModule},
+    "storage_module": {"local": StorageModule, "blob": StorageModule},
```

Comment:
```suggestion
    "service_module": {"default": ServiceModule, "local": ServiceModule, "echo": ServiceModule},
    "state_machine_module": {"default": StateMachineModule},
    "storage_module": {"default": ServiceModule, "local": StorageModule, "blob": StorageModule},
```

Reply:
You should have defaults for the modules that may be interchanged. When they are added with multiple modules the `local` will be the default one for these two for example.

Reply:
Implemented default selections for all modules.